app/controllers/admin/take_part_pages_controller.rb

```diff
@@ -53,4 +55,20 @@ def take_part_page_params
       :title, :summary, :body, :image, :image_alt_text, :image_cache
     )
   end
+
+  def patch_getinvolved_page_links
+    get_involved_cid = "dbe329f1-359c-43f7-8944-580d4742aa91"
```

Comment:
A little nitpick, but could `cid` be fully spelt out as `content_id` to keep the naming consistent with other places.
```suggestion
    get_involved_content_id = "dbe329f1-359c-43f7-8944-580d4742aa91"
```